frontend/src/v5/store/users/users.selectors.ts

```diff
@@ -30,5 +30,5 @@ export const selectUser = createSelector(
 	(_, teamspace) => teamspace,
 	(_, userName) => userName,
 	(state, teamspace, userName): IUser | null => (state.usersByTeamspace[teamspace] || [])
-		.find((teamspaceUser) => teamspaceUser.user === userName),
+		.find((teamspaceUser) => teamspaceUser.username === userName),
```

Comment:
This creates a bug, when clicking on the revisions, the page goes blank . There are no tests for users.  We need to add tests for these, I already created the issue #3073

Reply:
change reverted